app/views/admin/whats_new/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :page_title, t('whats_new.title') %>
+<% content_for :title, t('whats_new.title') %>
+<% content_for :title_margin_bottom, 0 %>
+
+<%
+  sections = [
+    t('whats_new.introduction'),
+    t('whats_new.recent_changes'),
+    t('whats_new.guidance'),
+  ]
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body-l"><%= t('whats_new.summary') %></p>
+
+    <p class="govuk-body app-view-whats-new__last-updated"><%= t('whats_new.last_updated') %></p>
+
+    <div id="content-list">
+      <%= render "govuk_publishing_components/components/contents_list", {
+        contents: sections.map do | section |
+          {
+            href: "##{section[:heading].parameterize(separator: '-')}",
+            text: section[:heading],
+          }
+        end
+      } %>
+    </div>
+
+    <% sections.each do | section | %>
+      <%= render partial: "admin/whats_new/section", locals: section %>
+    <% end %>
```

Comment:
Since we only have one section this loop seems premature. We should just embed the section code directly here. It's only a few lines and saves confusion of a loop of 1.

Reply:
done